web/thesauruses/c/17/functions.json

```diff
@@ -0,0 +1,72 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "functions"
+    },
+    "concepts": {
+        "void_functions_no_parameters": {
+            "code": "void foo() { return; }",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and takes no parameters"
+        },
+        "void_function_with_parameters": {
+            "code": "void foo(char c, int i, double d) { return; }",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and that takes 1 or more defined parameters"
+        },
+        "void_function_with_keyword_parameters": {
+            "code": "unknown",
+            "name": "Function that does not return a value and that takes 0 or more defined keyword parameters"
+        },
+        "void_function_variable_parameters": {
+            "code": "unknown",
+            "comment": "a header file <stdargs.h> is available",
```

Comment:
This doesn't help a user that's trying to look up the answer. If the answer is in that header file, write an example of how it works.